File: src/main/java/com/congwiny/basic/ExtAttrUsing.java
package com.congwiny.basic;

import text.ExtAttrStringBuilderKt;

public class ExtAttrUsing {
    public static void main(String[] args) {
        //Java 访问扩展属性时，应显式地调用它的getter函数
        char c = ExtAttrStringBuilderKt.getLastChar2(new StringBuilder("abc"));
        System.out.println(c);
    }
}

File: src/main/kotlin/com/congwiny/basic/ExtAttrUsing.kt
package com.congwiny.basic

import text.lastChar2

fun main(args: Array<String>) {
    val sb = StringBuilder("Kotlin")
    //像成员一样访问它
    println(sb.lastChar2) //n

    val sb2 = StringBuilder("Kotlin?")
    sb2.lastChar2 = '!'
    println(sb2) //Kotlin!
}